worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -143,6 +143,12 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
       if (operator != null) {
         operator.shutdown()
       }
+    } else if (diskStatus == DiskStatus.READ_OR_WRITE_FAILURE) {
+      logInfo(s"Face read or write failure, destroy all file writers under ${mountPoint}")
+      // TODO:
+      //  1. do we need to wait rest file workers to finish their work(like close()?) or just immediately destroy them
+      //  because there is existed write error
+      workingDirWriters.get(mountPoint).asScala.foreach(_.destroy)
```

Comment:
Why not we destroy it in FileWirter's `notifyError`?

Reply:
Also, destroy should pass a destroy reason, normal destroy or destroy caused by device error?